### ISSUE TITLE
Code doesn't work? (verify device login)

### ISSUE THREAD

**Issue by hujiaxin1029** [nayrk/Blink#7]
Hi Ryan,

I got the issue that after running the code it mentioned all the videos have been downloaded but it's not...
Here is what I did:
1. Modify the code by changing to my login name: 
![image](https://user-images.githubusercontent.com/55061186/83949795-930c7e00-a7da-11ea-922d-37340cecad08.png)
2. After running the code for 2 mins, here is what I got: 
![image](https://user-images.githubusercontent.com/55061186/83949816-b20b1000-a7da-11ea-8216-4cf266de2721.png)
3. But when I opened the folder, there is nothing....
![image](https://user-images.githubusercontent.com/55061186/83949828-c818d080-a7da-11ea-8f98-1b4f93225b4a.png)

4. I checked my blink email and it shows verify my device, but I don't know how to verify since I already logged in on my phone, anyone can help with this? Thanks!
![image](https://user-images.githubusercontent.com/55061186/83949863-057d5e00-a7db-11ea-975b-e38a1144143c.png)

**Comment by sutty550e** [comment]
Something isn't right, somehow, because the folder that it says it has downloaded to is not where you are looking, because it makes a folder called Blink anyway, so you should be looking in Blink\Blink, but of course you can't go any deeper than that, because we can see there is no Blink folder in the Blink folder you instructed it to create, in your path line.

Mine doesn't, as yet, trigger the response to input a PIN. You may have seen from the issue thread you first posted in, that the script briefly stopped working, until a fix was discovered by Colinreid89. Ryan, nayrk, updated his script to incorporate this change.

It probably stopped working because they implemented the two step verification process for the app, and the fix was to point it to a different log in location, but really that still allows login without two step verification.

I just used mine, and received no email, yet you did, so it might not be accepting your login fully, so it isn't allowed to download any videos.

Maybe, and this is just a guess, have you created an IFTTT account, and logged into the Blink service there? This used to be detailed in the script as a prerequisite for success. Certainly can't hurt to try. If you're not clear what I'm talking about, it should be detailed in the history of this github project, but do post back if you're stuck.

**Comment by hujiaxin1029** [comment]
Thanks for the reply, I just created an IFTTT account and activated my blink service there (I didn't know this was required previously.), but still have the same problem that my email still receive the verification PIN. Did I miss anything else? Thanks

**Comment by sutty550e** [comment]
I'm sorry I can't help. Mine does not trigger the response to send me a code, so I don't really have anything else to offer. Maybe they are rolling it out to more and more users, and eventually it will stop working for all. It wouldn't surprise me if this is what's happening, because the change that made it work again, didn't create/allow for the entry and transmission of the verification code, so it wasn't really complying with their stated new aim of two step verification. I'd been thinking similar, and then Ryan mentioned it to me in an email to me, so he too must have been wondering about it. Did IFTTT require a verification code when you linked the Blink service? If not, maybe they have a dispensation.

Last thing, you could actually try making a Blink IFTTT applet, and causing it to run, just so it might open up something that was previously closed off to you. Just clutching at straws here. I hope someone else can help.

**Comment by hujiaxin1029** [comment]
The IFTTT doesn't require a verification code. However I still received the same issue (verification PIN sent to my email) and can't download any videos using the code. I also tried to make a Blink IFTTT applet and still the same..

**Comment by sutty550e** [comment]
Okay, it was a shot in the dark. Hopefully some of the other, more knowledgeable guys, will be able to come up with better suggestions.

**Comment by hujiaxin1029** [comment]
Thanks for the help tho!
Still looking for some other folks who can help here.

**Comment by locvez** [comment]
@nayrk - I have 2FA enabled and the code works fine, I receive videos in the expected folders and I have a scheduled task setup to execute this script every hour. However every time this runs I get an email with the 2FA code to verify the app but can find no way to verify using the script.

Can you advise if it is possible to verify this script somehow to prevent the email spam?

**Comment by BroomofBronx** [comment]
I have noticed that the Blink Minis cameraclips doesn´t download. I haven't done much programing in my days, but a little for a long time ago. I managed to add Write-Host, now I can see that the code finds all of my three networkNames CMK, CMK2 and CMK3. In two of them the code finds all the cameras and downloads the clips, flawless. ;)
BUT the Blink Minis networkName doesn't get created as a folder on my harddrive and the code doesn't find the camera (the Blink Mini itself). Does the Mini need some other code?
Would be nice if someone could help me. Thanks.

![ps](https://user-images.githubusercontent.com/67604641/86047287-f80c6a00-ba4e-11ea-8b81-4d5467c4ea33.png)
